docs/usage/low-level-stac-ingestion-eoapi.md

```diff
@@ -0,0 +1,52 @@
+# Low-level STAC data ingestion into eoAPI
+
+In addition to higher-level and user-friendly interfaces, there is a basic mechanism for pushing STAC data into eoAPI's PgSTAC database.
+
+## Use the ingestion script and Makefile
+
+eoAPI-k8s includes a [script](https://github.com/developmentseed/eoapi-k8s/blob/960435300f04e8742e71d22ff4873ef95d60dad7/ingest.sh) which is also
+referenced in the project's [Makefile](https://github.com/developmentseed/eoapi-k8s/blob/960435300f04e8742e71d22ff4873ef95d60dad7/Makefile), for convenience.
```

Comment:
I would suggest to link to them on the `main` branch to always link to the ones most up to date.

Reply:
Yeah, I used permanent links to make sure they do not break if we change things around in eoapi-k8s, which I think is likely enough.

But let's do main anyways for now.